libs/react-helpers/src/hooks/use-data-provider.ts

```diff
@@ -55,7 +56,7 @@ export function useDataProvider<Data, Delta>({
     return Promise.reject();
   }, []);
   const callback = useCallback(
-    ({ data, error, loading, delta, insertionData, totalCount }) => {
+    ({ data, error, loading, delta, insertionData, totalCount }: any) => {
```

Comment:
non-blocking: this `any` doesn't look nice, maybe it's possible to create from here an issue to fix this in the future?

Reply:
or this proposal maybe can handle it: https://github.com/vegaprotocol/frontend-monorepo/pull/977

Reply:
@macqbat It looks like we started working on this as the same time, I pushed fix already